### DISCUSSION TITLE
Without rust

### DISCUSSION THREAD

**Discussion by Appelg** [solokeys/solo2#136] [General]
Can you please fix a way to update the keys without having to install the rust runtime/cargo cr*p on my system? I just want a .exe to run and then delete. Thanks.

**Comment by needs-coffee** [comment]
@Appelg Just download the exe from the releases page on [the solo2-cli repo](https://github.com/solokeys/solo2-cli/releases/tag/v0.2.1) and run this from CMD/powershell - rust not required

**Reply by Appelg** [reply]
Thanks. Since https://solokeys.com/pages/firmware-update-for-windows shows the full blown install and Cargo, I figured there was no built exe.

Why not just link the exe from that info page instead of the current method?

**Reply by needs-coffee** [reply]
Beats me - I agree it makes more sense to guide most users to use the pre compiled versions and use cargo if they need it compiled for a more niche platform.
